docs/topics/clusterdefinitions.md

```diff
@@ -910,6 +910,8 @@ https://{keyvaultname}.vault.azure.net:443/secrets/{secretName}/{version}
 | enableAutomaticUpdates        | no       | If set to `true` Windows Update will be configured to automatically apply updates on Windows nodes. Default: `false`                                                                                                                                            |
 | enableCSIProxy                | no       | If set to `true` the csi-proxy specified by `windowsProfile.csiProxyURL` will get installed during node provisioning. See [Windows Csi Proxy](csi-proxy-windows.md) for more details.                                                                           |
 | provisioningScriptsPackageURL | no       | Path to a package containing provisioning scripts found in [/staging/provisioning/windows](/staging/provisioning/windows). Default values will be maintained by aks-engine team and contain script content signed by Microsoft.                                 |
+| windowsPauseImageURL          | no       | Path to a Windows pause image. Default values will be maintained by aks-engine team.                                 |
+| alwaysPullWindowsPauseImage   | no       | If set to `true`, it will always pull the Windows pause image from MCR repo. Default: `false`                                   |
```

Comment:
it will always pull but not nessarilly from the MCR repo.  It would always pull from the repo specified.